docs/manualGrading/index.md

```diff
@@ -11,11 +11,11 @@ For questions that require manual grading, the question can be set up for manual
 
 ```json
 {
-    "questions": [
-        {"id": "manualGradingOnly", "manualPoints": 2},
-        {"id": "combinedManualAndAutoGrading", "autoPoints": 2, "manualPoints": 6}
-    ]
-    ...
+  "questions": [
+    { "id": "manualGradingOnly", "manualPoints": 2 },
+    { "id": "combinedManualAndAutoGrading", "autoPoints": 2, "manualPoints": 6 }
+  ]
+  // ...
```

Comment:
We could probably drop most of these `...` lines. They were intended to show that this is not a complete `assessmentInfo.json` file, but elsewhere we are showing partial files without any special annotation.